### PR TITLE
[5.0 benchmark definitions] do not merge until 5.0

### PR DESCRIPTION
**Pull request by IcarusTwine**
This is a preemptive PR just so i dont lose it and in case the benchmark is correct for some sheets so we dont rush about to fix things.

### CODE REVIEW

File: SaintCoinach/ex.json [hunk 1]
```diff
@@ -11609,6 +11698,14 @@
...
           "index": 9,
           "name": "HuntingLogExpReward"
+        },
+        {
+          "index": 11,
+          "name": "ItemLevelSync"
+        },
+        {
+          "index": 14,
+          "name": "cLevel"
```
Comment (vekien) [inline SaintCoinach/ex.json:11708]:
```suggestion
          "name": "ClassJobLevel"
```

Is this "ClassJob" ?

**Comment by ufx** [comment]
Great work on figuring the changes to this.  Since ex.json will be split by the time 5.0 is out, we'll have to redo this in the new files.  Leaving the do-not-merge label so we can come back to it then.

File: SaintCoinach/ex.json (hunk 1 above)
Comment (IcarusTwine) [reply SaintCoinach/ex.json:11708]: supposedly crafterlevel?, apparently a level for crafting that is used to determine potency penalties/bonuses

Comment (vekien) [reply SaintCoinach/ex.json:11708]: Maybe CraftingLevel? C is too ambiguous.

**Comment by IcarusTwine** [comment]
closed and reopened for the split at #233
@viion named that column CraftingLevel to avoid confusion.

**Pull request closed without merging**